module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -625,7 +625,9 @@ public function getMyProfile($patron)
             'zip' => $profile->personal->addresses[0]->postalCode ?? null,
             'phone' => $profile->personal->phone ?? null,
             'mobile_phone' => $profile->personal->mobilePhone ?? null,
-            'expiration_date' => $profile->expirationDate ?? null,
+            'expiration_date' => $this->dateConverter->convertToDisplayDate(
+                "Y-m-d H:i", $profile->expirationDate
+            ) ?? null,
```

Comment:
I don't think `?? null` works here... that's to ensure PHP doesn't throw an "undefined" error about `$profile->expirationDate`, but now the code is assuming that `$profile->expirationDate` will be set. Might be cleaner to do:

<pre>
$expiration = isset($profile->expirationDate)
    ? $this->dateConverter->convertToDisplayDate("Y-m-d H:i", $profile->expirationDate)
    : null;
</pre>

And then assigning the `$expiration` value in the return array...

Reply:
Thanks! applied this suggestion.